cmd/nerdctl/container_commit_test.go

```diff
@@ -53,3 +56,56 @@ func TestCommit(t *testing.T) {
 		base.Cmd("rmi", testImage).Run()
 	}
 }
+
+func kubeCtlHelper(base *testutil.Base, args ...string) *testutil.Cmd {
+	base.T.Helper()
+	icmdCmd := icmd.Command("kubectl", args...)
+	icmdCmd.Env = base.Env
+	return &testutil.Cmd{
+		Cmd:  icmdCmd,
+		Base: base,
+	}
+}
+
+func TestKubeCommitPush(t *testing.T) {
+	t.Parallel()
+
+	base := testutil.NewBaseForKube(t)
+
+	var containerID string
+
+	setup := func() {
+		kubeCtlHelper(base, "run", "--image", "debian", "my-app", "--", "sleep", "Inf").
+			AssertOK()
+
+		kubeCtlHelper(base, "wait", "po", "my-app", "--for=condition=ready", "--timeout=1m").
```

Comment:
```suggestion
		kubeCtlHelper(base, "wait", "pod", "my-app", "--for=condition=ready", "--timeout=1m").
```

Reply:
Afaik both syntax work? Anyhow, changed to `pod` (better, more explicit).
Thanks.